package.json

```diff
@@ -28,18 +28,20 @@
     "test": "jest --config .jestrc",
     "test:ci": "jest --config .jestrc --runInBand",
     "test:watch": "npm run test -- --watch",
-    "watch": "npm run build:react -- --watch"
+    "watch": "npm run build:react -- --watch",
+    "postinstall": "npm run yogafix",
+    "yogafix": "node ./yogafix"
   },
   "dependencies": {
-    "css-layout": "^1.1.1",
     "error-stack-parser": "^2.0.0",
     "invariant": "^2.2.2",
     "murmur2js": "^1.0.0",
     "normalize-css-color": "^1.0.1",
     "prop-types": "^15.5.8",
     "sketch-constants": "^1.0.2",
     "sketchapp-json-flow-types": "^0.2.0",
-    "sketchapp-json-plugin": "^0.0.3"
+    "sketchapp-json-plugin": "^0.0.3",
+    "yoga-layout": "^1.6.0"
```

Comment:
Probably safer to pin the dependency if we are to postinstall modify it

Reply:
Good call. Fixed.